c50078320.lua

```diff
@@ -37,15 +37,15 @@ function c50078320.operation(e,tp,eg,ep,ev,re,r,rp)
 	Duel.Hint(HINT_SELECTMSG,tp,HINTMSG_CODE)
 	if bit.band(cv,ANNOUNCE_CARD)~=0 then
 		--c:IsType(cv) and not c:IsCode(code)
-		ac=Duel.AnnounceCardFilter(tp,cv,OPCODE_ISTYPE,code,OPCODE_ISCODE,OPCODE_NOT,OPCODE_AND)
+		ac=Duel.AnnounceCard(tp,cv,OPCODE_ISTYPE,code,OPCODE_ISCODE,OPCODE_NOT,OPCODE_AND)
```

Comment:
this card need more changes

Reply:
Why? _if changes are needed that is because it was wrong in first place then_

Reply:
`ANNOUNCE_CARD` or `ANNOUNCE_CARD_FILTER` should be obsolete

Reply:
In fact I removed AnnounceCardFilter, and merged the 2 functions